src/api/appUsers.js

```diff
@@ -24,6 +24,30 @@ export class AppUsersApi extends BaseApi {
     });
   }
 
+  create(userId, props = {}) {
+    if (!userId || !userId.trim()) {
+      throw new Error('Must provide an user id.');
+    }
+
+    let payload = Object.assign({
+      userId: userId
+    }, props);
+
+    if (props.signedUpAt instanceof Date) {
```

Comment:
JSON serialization will do this for you. In fact, you should reject a signedUpAt param if it isn't a `Date`